pkg/project/project.go

```diff
@@ -98,8 +100,18 @@ func (p *Project) CommonSetupSteps() error {
 			},
 		}}
 
+	newNotify := func(operationName string) func(error, time.Duration) {
+		return func(err error, delay time.Duration) {
+			log.Printf(fmt.Sprintf("%s failed, retrying with delay %.0fm%.0fs: '%#v'", operationName, delay.Minutes(), delay.Seconds(), err))
```

Comment:
The project here has a logger. Lets please use it. The duration we get implements stringer. Lets use this as well.

Reply:
Good point, using the logger member. About the duration, you mean using the `%s` verb with it as a whole, without treating minutes and seconds separately?

Reply:
Yeah, we can also just call `String()` so the formatter does not have to do it, but this is minor. My main point is that the formatting done by the stringer is simpler to use and its output easier to read because it omits minutes and hours if the actual duration is just about seconds.

Reply:
Makes sense, done.